contrib/ltree/expected/ltree.out

```diff
@@ -7823,11 +7823,10 @@ SELECT * FROM ltreetest WHERE t ? '{23.*.1,23.*.2}' order by t asc;
 drop index tstidx;
 create index tstidx on ltreetest using gist (t gist_ltree_ops(siglen=0));
 ERROR:  value 0 out of bounds for option "siglen"
-DETAIL:  Valid values are between "1" and "2024".
+DETAIL:  Valid values are between "1" and "8168".
 create index tstidx on ltreetest using gist (t gist_ltree_ops(siglen=2025));
```

Comment:
Should we change the test to get the same error as in the original?
```suggestion
create index tstidx on ltreetest using gist (t gist_ltree_ops(siglen=8169));
```

Reply:
reworked